src/views/EnigmaPopup/EnigmaPopup.vue

```diff
@@ -31,10 +57,27 @@
   export default {
     name: 'EnigmaPopup',
     components: { Button, Icon },
+    data() {
+      return { response: '' };
+    },
     computed: {
+      currentUser() {
+        return this.$store.state.user;
+      },
       enigma() {
         return this.$store.state.app.enigmaPopup.enigma;
       },
+      expectedReponse() {
+        return this.$store.state.enigmas.find(e => e.id === this.enigma.id).response;
+      },
+      canUserEnterResponse() {
+        if (this.expectedReponse) return true;
```

Comment:
J'ai un doute ici, si on a `this.expectedReponse` a `true` il ne faudrait pas que l'utilisateur puisse saisir une réponse ni avoir accès au bouton valider non? Normalement dès qu'on trouve qu'il y a une réponse dans les énigmes, c'est que la solution a été apportée et donc c'est la fin de l'énigme non?  
Ce n'est pas plutôt ça  `if !(this.expectedReponse) return true; ?`